index.html

```diff
@@ -3737,55 +3739,53 @@ <h2>DID Resolution</h2>
 be a conformant <a>DID</a> as defined in <a href="#did-syntax"></a>.
             </dd>
             <dt>
-did-resolution-input-metadata
+options
             </dt>
             <dd>
-A <a href="#metadata-structure">metadata structure</a> consisting of input
-options to the <code>resolve</code> and <code>resolveRepresentation</code>
-functions in addition to the <code>did</code> itself. Properties defined by this
-specification are in <a href="#did-resolution-input-metadata-properties"></a>.
-This input is REQUIRED, but the structure MAY be empty.
+A <a href="#metadata-structure">metadata structure</a> containing properties
+defined in <a href="#did-resolution-options"></a>. This input is
+REQUIRED, but the structure MAY be empty.
             </dd>
         </dl>
 
         <p>
-The output variables of these functions are as follows:
+The return value of <code>resolve</code> is the
+<a>didResolutionMetadata</a>, <a>didDocument</a>, and
+<a>didDocumentMetadata</a> properties. The return value of
+<code>resolveRepresentation</code> is the
+<a>didResolutionMetadata</a>, <a>didDocumentStream</a>, and
+<a>didDocumentMetadata</a> properties. These properties are described below:
```

Comment:
These are not properties, which implies that they're part of a structure, they are separate return values. Trying to straighten that out here:

```suggestion
These functions each return multiple values, and no limitations
are placed on how these values are returned together.
The return values of <code>resolve</code> are
<a>didResolutionMetadata</a>, <a>didDocument</a>, and
<a>didDocumentMetadata</a>. The return values of
<code>resolveRepresentation</code> are
<a>didResolutionMetadata</a>, <a>didDocumentStream</a>, and
<a>didDocumentMetadata</a> properties. These values are described below:
```

Reply:
+1 to not using the term "properties" here. @jricher in your suggestion you removed "properties" in once sentence, but you forgot to also remove it in the other sentence.

Reply:
@jricher - I think you missed deleting ` properties` from line 3759? (Someday, Github will figure out that suggestions might be needed on suggestions...)